docs/scenarios/library-authoring/trimming.md

```diff
@@ -0,0 +1,177 @@
+
+To trim this hypothetical library, you would start with the `project.json` file.
+
+```json
+{
+    "dependencies":{
+        "NETStandard.Library":"1.5.0-rc2"
+    },
+    "frameworks": {
+        "netstandard1.0": {}
+     }
+}
+```
+
+Next, you would restore packages with `$ dotnet restore`, inspect the `project.lock.json` file, and find all the packages restore for `NETSTandard.Library`.
+
+Here's the relevant section in the `project.lock.json` file:
+
+```json
+"NETStandard.Library/1.5.0-rc2-24027": {
+    "type": "package",
+    "dependencies": {
+        "Microsoft.NETCore.Platforms": "1.0.1-rc2-24027",
+        "Microsoft.NETCore.Runtime": "1.0.2-rc2-24027",
+        "Microsoft.Win32.Primitives": "4.0.1-rc2-24027",
+        "System.AppContext": "4.1.0-rc2-24027",
+        "System.Collections": "4.0.11-rc2-24027",
+        "System.Collections.Concurrent": "4.0.12-rc2-24027",
+        "System.Console": "4.0.0-rc2-24027",
+        "System.Diagnostics.Debug": "4.0.11-rc2-24027",
+        "System.Diagnostics.Tools": "4.0.1-rc2-24027",
+        "System.Diagnostics.Tracing": "4.1.0-rc2-24027",
+        "System.Globalization": "4.0.11-rc2-24027",
+        "System.Globalization.Calendars": "4.0.1-rc2-24027",
+        "System.IO": "4.1.0-rc2-24027",
+        "System.IO.Compression": "4.1.0-rc2-24027",
+        "System.IO.Compression.ZipFile": "4.0.1-rc2-24027",
+        "System.IO.FileSystem": "4.0.1-rc2-24027",
+        "System.IO.FileSystem.Primitives": "4.0.1-rc2-24027",
+        "System.Linq": "4.1.0-rc2-24027",
+        "System.Net.Http": "4.0.1-rc2-24027",
+        "System.Net.Primitives": "4.0.11-rc2-24027",
+        "System.Net.Sockets": "4.1.0-rc2-24027",
+        "System.ObjectModel": "4.0.12-rc2-24027",
+        "System.Reflection": "4.1.0-rc2-24027",
+        "System.Reflection.Extensions": "4.0.1-rc2-24027",
+        "System.Reflection.Primitives": "4.0.1-rc2-24027",
+        "System.Resources.ResourceManager": "4.0.1-rc2-24027",
+        "System.Runtime": "4.1.0-rc2-24027",
+        "System.Runtime.Extensions": "4.1.0-rc2-24027",
+        "System.Runtime.Handles": "4.0.1-rc2-24027",
+        "System.Runtime.InteropServices": "4.1.0-rc2-24027",
+        "System.Runtime.InteropServices.RuntimeInformation": "4.0.0-rc2-24027",
+        "System.Runtime.Numerics": "4.0.1-rc2-24027",
+        "System.Text.Encoding": "4.0.11-rc2-24027",
+        "System.Text.Encoding.Extensions": "4.0.11-rc2-24027",
+        "System.Text.RegularExpressions": "4.0.12-rc2-24027",
+        "System.Threading": "4.0.11-rc2-24027",
+        "System.Threading.Tasks": "4.0.11-rc2-24027",
+        "System.Threading.Timer": "4.0.1-rc2-24027",
+        "System.Xml.ReaderWriter": "4.0.11-rc2-24027",
+        "System.Xml.XDocument": "4.0.11-rc2-24027"
+    }
+}
+```
```

Comment:
I get a much shorter list for netstandard1.0, I think this list is for a higher version of netstandard.

Shouldn't this whole guide be consistent, so that those who follow it at home, see exactly what the guide says they should see?